docs/ACCESS_PATTERNS.md

```diff
@@ -0,0 +1,34 @@
+| a trial session record                                     | trial-session\|${TRIAL_SESSION_ID}              | trial-session\|${TRIAL_SESSION_ID}       | trial-session-catalog                       |
+| copy the trial session and attach it to the user           | trial-session-working-copy\|${TRIAL_SESSION_ID} | user\|${USER_ID}                         |                                             |
+| add a case not to the user                                 | user-case-note\|${USER}                         | user\|${USER_ID}                         |                                             |
+| create a user                                              | user\|${USER_ID}                                | user\|${USER_ID}                         |                                             |
+| lookup a user by their email                               | user-email\|${EMAIL}                            | user\|${USER_ID}                         |                                             |
+| for getting all users in a section                         | section\|${SECTION_NAME}                        | user\|${USER_ID}                         |                                             |
+| work item saved in a section                               | section\|${SECTION_NAME}                        | work-item\|${WORK_ITEM_ID}               | work-item\|${WORK_ITEM_ID}                  |
+| save outbox workitems for a section sorted by date         | section-outbox\|                                | $datetime                                | work-item\|${WORK_ITEM_ID}                  |
+| save work items for a user inbox                           | user\|${USER_ID}                                | work-item\|${WORK_ITEM_ID}               | work-item\|${WORK_ITEM_ID}                  |
+| user outbox workitems sorted by date                       | user-outbox\|${USER_ID}                         | $datetime                                | work-item\|${WORK_ITEM_ID}                  |
+| a work item entry                                          | work-item\|${WORK_ITEM_ID}                      | work-item\|${WORK_ITEM_ID}               | work-item\|${WORK_ITEM_ID}                  |
+| a work item on a case                                      | case\|${CASE_ID}                                | work-item\|${WORK_ITEM_ID}               |                                             |
+| associate practitioner by name for lookup                  | privatePractitioner\|${NAME}                    | user\|${userId}                          |                                             |
+| associate practitioner by bar number for lookup            | privatePractitioner\|${BAR_NUMBER}              | user\|${userId}                          |                                             |
```

Comment:
Love the list. I just wonder if you could sort by PK. I think everything falls into order better for my 🧠